projects/packages/publicize/src/rest-api/class-jetpack-social-controller.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * The Jetpack Social Controller class.
+ *
+ * @package automattic/jetpack-publicize
+ */
+
+namespace Automattic\Jetpack\Publicize\REST_API;
+
+use Automattic\Jetpack\Connection\Traits\WPCOM_REST_API_Proxy_Request;
+use Automattic\Jetpack\Publicize\Publicize_Utils as Utils;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * Jetpack Social Controller class.
+ */
+class Jetpack_Social_Controller extends Base_Controller {
+
+	use WPCOM_REST_API_Proxy_Request;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->base_api_path = 'wpcom';
+		$this->version       = 'v2';
+
+		$this->namespace = "{$this->base_api_path}/{$this->version}";
+		$this->rest_base = 'jetpack-social';
```

Comment:
As mentioned above, let use a specific name like `publicize/shares-data` as the `$rest_base`

Reply:
Let's talk about this on the other thread